KicadModTree/nodes/specialized/FilledRect.py

```diff
@@ -28,9 +28,9 @@ class FilledRect(Node):
         See below
 
     :Keyword Arguments:
-        * *start* (``Point``) --
+        * *start* (``Vector``) --
```

Comment:
Type is actually Vector2D when I'm right

Reply:
We did not specify the vector dimensions in any docstring. Fixed that (all instances are vector2d except the 3d model stuff)